bootstrap.py

```
@@ -0,0 +1,256 @@
+                break
+    finally:
+        stop_services(services)
+        print("모든 서비스를 종료했습니다.")
+
+
+if __name__ == "__main__":
+    try:
+        run()
+    except KeyboardInterrupt:
+        print("\n사용자 인터럽트로 종료합니다.")
+        raise SystemExit(130)
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)